src/lib/style/abstracts/_mixins.scss

```diff
@@ -194,3 +194,35 @@ $link-underline-color-default: adjust-color(
     @include disabled($ignoreLink: true);
   }
 }
+
+// Always top margins to the MuiTypography variants causes problems with some built-in MUI components
+// Use this mixin within a selector to apply the margins
+@mixin MuiTypographyTopMargins() {
```

Comment:
I am not sure if we should use this mixin when developing new components, but it seemed to be the easiest way to apply the margin change to the handful of components that relied on them.